dcc-portal-ui/app/scripts/projects/views/project.summary.html

```diff
@@ -15,8 +15,9 @@ <h3>
                     <translate>Name</translate>
                 </th>
                 <td>
-                    <a href="http://icgc.org/node/{{ProjectCtrl.project.icgcId}}" target="_blank">
-                        <i class="icon-external-link"></i>{{ProjectCtrl.project.name}}</a>
+                    <a href="#" data-ng-click="ProjectCtrl.openCGPNodeModal(ProjectCtrl.project.icgcId)">
```

Comment:
opens the project profiles in a modal, rather than linking to their old location